mpas_analysis/shared/climatology/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .climatology import get_lat_lon_comarison_descriptor, get_remapper, \
+    get_mpas_climatology_file_names, get_observation_climatology_file_names, \
+    compute_monthly_climatology, compute_climatology, cache_climatologies, \
+    update_start_end_year, add_years_months_days_in_month, \
+    remap_and_write_climatology
```

Comment:
I see: this must be the explanation to my confusion above. I think here you meant: 

> This allows us to use `from ..shared.climatology import ...` instead of `from ..shared.climatology.climatology import ...`
?

One word of caution though: we need to change any other script that now uses `from ..shared.climatology.climatology import ...`. I am pretty sure both meridional_overturning_circulation and meridional_heat_transport do that.